tasks/elasticsearch-config.yml

```diff
@@ -24,6 +24,18 @@
   template: src=elasticsearch.j2 dest={{ default_file }} mode=0644 force=yes
   notify: restart elasticsearch
 
+#Copy the systemd specific file if systemd is installed
+- name: Make sure destination dir exists
+  file: path={{ sysd_config_file | dirname }} state=directory recurse=yes mode=0755
+  
+- name: Copy Systemd File for Instance
+  become: yes
+  copy: src=systemd/elasticsearch_override.conf dest={{ sysd_config_file }} mode=0644 force=yes
```

Comment:
Maybe we should add [`blockinfile`](https://docs.ansible.com/ansible/latest/modules/blockinfile_module.html#blockinfile-module) instead of `copy` to not override other configuration that some users could add manually to this file.

Reply:
I think I found a better solution using ini_file module

This task below will :

- create the file with only this setting if the file doesn't exist
- make sure the setting is defined in the Service section if the file already exists

`ini_file: path={{ sysd_config_file }} section=Service option=LimitMEMLOCK value=infinity mode=0644` 

What do you think about it?

Reply:
Awesome, I didn't know `ini_file` module.

Reply:
Fixed with 5bac5a4